mcs/class/corlib/System/TimeZoneInfo.cs

```diff
@@ -116,11 +116,11 @@ static TimeZoneInfo CreateLocal ()
 
 			try {
 				return FindSystemTimeZoneByFileName ("Local", "/etc/localtime");	
-			} catch {
+			} catch (TimeZoneNotFoundException) {
```

Comment:
the question is: 
- if `/etc/localtime` is present, but invalid - shall we abort and notify the user (fail fast), or 
- shall we continue searching in `/usr/share/zoneinfo/localtime` ?